process-engine/identity_server
# Add docker support

## What did you change?

This PR adds docker support.

To avoid missing certificates and since this is solely for demonstration purposes, I've set this environment variable:

```
ENV ASPNETCORE_ENVIRONMENT Development
```

## How can others test the changes?

The most interesting part about this PR is the ability to set the environment variables `MICROSOFT_CLIENT_ID` and `MICROSOFT_CLIENT_SECRET` externally.

If you set them correctly and run the docker image, you should see `Microsoft` as an external login when you visit `http://localhost:{{YOUR_PORT}}/account/login?returnUrl=%2Fgrants`.

Please also check whether or not you can log in using the external login.

On successful login you should see your name on the top left and a dropdown menu with a `Logout` option.

The resulting docker image has also been published to [Docker Hub](https://hub.docker.com/r/5minds/processengine-identity_server/).

## PR-Checklist

Please check the boxes in this list after submitting your PR:
